### PR TITLE
APP-175: [Objective-C] Basic level Apps: Credentials management

### PR DESCRIPTION
**Pull request by am1kk**
http://jira.kaaproject.org/browse/APP-175

### CODE REVIEW

**Comment by am1kk** [comment]
@vilimets Review please.

File: credentialsdemo/source/objc/CredentialsDemo/ViewController.m [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+@implementation ViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+
+    [self addLogWithText:@"CredentialsDemo started"];
+    
+    self.logTextView.hidden = YES;
+    self.logTextView.alpha = 100;
```
Comment (vilimets) [inline credentialsdemo/source/objc/CredentialsDemo/ViewController.m]: UIView alpha value should be number in the range 0.0 to 1.0

**Review by vilimets — CHANGES_REQUESTED** [review]

File: credentialsdemo/source/objc/CredentialsDemo/ViewController.m (hunk 1 above)
Comment (am1kk) [reply credentialsdemo/source/objc/CredentialsDemo/ViewController.m]: Fixed.

**Comment by am1kk** [comment]
@vilimets re-review please.

**Review by vilimets — APPROVED** [review]
LGTM

**Review by vilimets — APPROVED** [review]
LGTM

**Comment by Slash32** [comment]
@kaa-leeroy test this please

**Review by Slash32 — APPROVED** [review]

**Pull request merged**